modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -318,36 +376,43 @@ payload.users.*user
 ]
 ----
 
-
 == Descendant Selector
 
-Returns the list of all children. This selector can be chained with any other selector directly without requiering any `.`. For example
-`payload..` returns all the child values and the value of its children and recursively. But it can also be chained with `payload..name` where it will select the name to all its descendants, or `payload..*name` where it will select all the names each descendant may have.
+The descendant selector returns a list of all children. You can directly chain
+this selector to any other selector without using a single `.`. For example,
```

Comment:
Probably need a different example, as `payload..name` vs. `payload..*name` return the same output when given the input below. Could we make the point and show different results using something like `payload..user` and `payload..*user` on a different input, one that adds a second set of `user` elements, for example 
```
<users>
  <user>
    <name>Weave</name>
    <user>
      <name>BAT</name>
      <user>
        <name>BDD</name>
      </user>
    </user>
  </user>
  <user>
    <name>Weave2</name>
    <user>
      <name>BAT2</name>
      <user>
        <name>BDD2</name>
      </user>
    </user>
  </user>
</users>
```

Reply:
In this case, `payload..user`  returns: 
```
[
  {
    "name": "Weave",
    "user": {
      "name": "BAT",
      "user": {
        "name": "BDD"
      }
    }
  },
  {
    "name": "BAT",
    "user": {
      "name": "BDD"
    }
  },
  {
    "name": "BDD"
  },
  {
    "name": "BAT2",
    "user": {
      "name": "BDD2"
    }
  },
  {
    "name": "BDD2"
  }
]
```

Reply:
And `payload..*user` returns this (which includes the `user` element with the "Weave2" value):  
```
[
  {
    "name": "Weave",
    "user": {
      "name": "BAT",
      "user": {
        "name": "BDD"
      }
    }
  },
  {
    "name": "Weave2",
    "user": {
      "name": "BAT2",
      "user": {
        "name": "BDD2"
      }
    }
  },
  {
    "name": "BAT",
    "user": {
      "name": "BDD"
    }
  },
  {
    "name": "BDD"
  },
  {
    "name": "BAT2",
    "user": {
      "name": "BDD2"
    }
  },
  {
    "name": "BDD2"
  }
]
```